index.d.ts

```diff
@@ -223,7 +223,7 @@ export interface ChalkInstance extends ChalkFunction {
 	readonly bold: this;
 
 	/**
-	Modifier: Emitting only a small amount of light.
+	Modifier: Make text lower opacity.
```

Comment:
It's not opacity, that's misleading. Also, it's not consistent across emulators, so we should note that. Perhaps something like...

```suggestion
	Modifier: Darken the text slightly. (Inconsistent across emulators; might do nothing)
```